packages/dev/inspector-v2/src/components/properties/materials/pbrMaterialProperties.tsx

```diff
@@ -5,6 +5,39 @@ import { BoundProperty } from "../boundProperty";
 import { Color3PropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/colorPropertyLine";
 import { SyncedSliderPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/syncedSliderPropertyLine";
 import { SwitchPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/switchPropertyLine";
+import { ChooseTexturePropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/chooseTexturePropertyLine";
+import { useProperty } from "../../../hooks/compoundPropertyHooks";
+import { usePropertyChangedNotifier } from "../../../contexts/propertyContext";
+import type { Scene } from "core/scene";
+
+/**
+ * Helper to bind texture properties without needing defaultValue
+ * @returns ChooseTexturePropertyLine component
+ */
+const BoundTextureProperty: FunctionComponent<{
+    label: string;
+    target: PBRMaterial;
+    propertyKey: keyof PBRMaterial;

```

Comment:
It may be possible to restrict this further to only be keys that are of type BaseTexture.